templates/ccng-config.lib.yml

```diff
@@ -81,15 +81,21 @@ logging:
   max_retries: 1
 
 logcache:
-  host: #@ "https://log-cache.{}".format(data.values.system_domain)
+  host: metric-proxy
```

Comment:
To clarify, `metric-proxy` the name of your new component's Kubernetes Service resource? Why omit the protocol as well?

Reply:
Yes, `metric-proxy` is the name of the `Service`. There are two reasons to leave it out: the protocol isn't needed to make this work, and the actual protocol is gRPC over mTLS. Having is connect to `metric-proxy:8080` is the simplest and clearest way to define this here.

Reply:
Great, thanks for clarifying!